node/src/components/consensus/protocols/highway.rs

```diff
@@ -333,6 +333,7 @@ impl<I: NodeIdT, C: Context + 'static> HighwayProtocol<I, C> {
         let vertex = vv.inner();
         if let (Some(value), Some(timestamp)) = (vertex.value(), vertex.timestamp()) {
             if value.needs_validation() {
+                self.log_proposal(vertex, "requesting proposal validation");
```

Comment:
With vertices having 100 deploys and 2500 transfers this can be too much. Can we log just the hash instead? We can always look it up later on Clarity or via node's RPC.